docs/installation.md

```diff
@@ -52,17 +54,33 @@ Also make sure that your Podfile deployment target is set to >= 13.0 at the top
 
 ```ruby
 platform :ios, '13.0'
+...
+# if you get erros because of missing react-native-google-maps files e.g. RCTConvert+AirMap.h, consider adding the following 2 statements 
+pod 'GoogleMaps'
+pod 'Google-Maps-iOS-Utils'
 ```
 
 Add the following to your Podfile above the `use_native_modules!` function and run `pod install` in the ios folder:
 
 ```ruby
+target 'casualjob' do
 # React Native Maps dependencies
 rn_maps_path = '../node_modules/react-native-maps'
 pod 'react-native-google-maps', :path => rn_maps_path
+...
+post_install do |installer|
+  ...
+  # if you get erros because of missing react-native-google-maps files e.g. RCTConvert+AirMap.h, consider adding the following section
+  installer.pods_project.targets.each do |target|
+    if target.name == 'react-native-google-maps'
+        target.build_configurations.each do |config|
+          config.build_settings['CLANG_ENABLE_MODULES'] = 'No'
+        end
+    end
+  end
```

Comment:
This seems like a fix for an issue that wasn't discussed. Feel free to open an issue regarding what is fixed here :)

Reply:
I think it fixes the following issues. If you don't agree, you can decline it. 
#4552, #4492, #3597

Reply:
Any news here?

Reply:
I have no interest in "declining it", but we need to have a more thorough discussion regarding why this fixes the issue.

If this fixes any issue, please open a new one, describe how and why this fixes the issue. Keep them coming! 👍